packages/cli/src/utils/election.ts

```diff
@@ -8,22 +11,35 @@ import { bitIsSet, parseBlockExtraData } from '@celo/utils/lib/istanbul'
 export class ElectionResultsCache {
   private readonly cache = new Map()
 
-  constructor(private readonly election: any, private readonly epochSize: number) {}
+  constructor(
+    private readonly kit: ContractKit,
+    private readonly isCel2: boolean,
```

Comment:
suggestion... move isCel2 to after electionWrapper since it will be a temporary argument

Reply:
this is a very very light suggestion. as its probably not any more difficult to move later vs now